lamindb/models/schema.py

```diff
@@ -245,36 +245,31 @@ def add(self, features: Feature | list[Feature]) -> None:
 
 
 class Schema(Record, CanCurate, TracksRun):
-    """Schemas of datasets such as the set of columns of a `DataFrame`.
+    """A schema is a feature set, such as the set of columns of a `DataFrame`.
 
-    A simple schema is a feature set such as the set of columns of a `DataFrame`.
-
-    A composite schema has multiple slots, e.g., for an `AnnData`, one schema for slot `obs` and another one for `var`.
-
-    A schema can also merely define abstract constraints or instructions for dataset validation & annotation.
```

Comment:
Why remove this last one (I get why you remove the first sentence).

Reply:
I was thinking about this several times. I ended up following my initial impression which was: This is a difficult to understand sentence and doesn't have enough practical value.

If you think that the message is important, I am happy to try to phrase it more simply so that it's more easily understood.